core/src/main/java/de/jplag/comparison/TokenValueMapper.java

```diff
@@ -0,0 +1,66 @@
+package de.jplag.comparison;
+
+import java.util.HashMap;
+import java.util.IdentityHashMap;
+import java.util.List;
+import java.util.Map;
+
+import de.jplag.SharedTokenType;
+import de.jplag.Submission;
+import de.jplag.SubmissionSet;
+import de.jplag.Token;
+import de.jplag.TokenType;
+import de.jplag.logging.ProgressBarLogger;
+import de.jplag.logging.ProgressBarType;
+
+/**
+ * Maps the tokens in a submission names to integers so they can be used by {@link GreedyStringTiling}. Each token type
+ * will be assigned a unique number. The token lists in that form can be queried by calling
+ * {@link TokenValueMapper#getTokenValuesFor(Submission)}.
+ */
+public class TokenValueMapper {
+    private final Map<TokenType, Integer> tokenTypeValues;
+    private final Map<Submission, int[]> tokenValueMap;
+
+    /**
+     * Crates the mapping for a single submission. This will also show the progress to the user using the
```

Comment:
Documentation is inaccurate, right? Also: typo.

Reply:
Good catch!